source/components/defining-a-component.md

```diff
@@ -1,5 +1,5 @@
-To define a component, create a template whose name starts with
-`components/`. To define a new component, `{{blog-post}}` for example,
+To define a component, create a template in the `components/` directory.
```

Comment:
I think it would be even more clear to say "in the `app/templates/components/` directory".